interface/src/app/analyze/analysis/analysisDetail.tpl.html

```diff
@@ -73,10 +73,62 @@
         <td>{{sb.sampleData[id].name}}</td>
         <td ng-repeat="at in analysis.exampleCols">{{
           sb.sampleData[id].annotations[at.typename]
-          }}</td>
+        }}</td>
```

Comment:
Same question as in line 42 here.

Reply:
See above. I agree this is a little funky but should make life much easier for CSS styling (and produces more correct HTML formatting).